dask_snowflake/tests/test_core.py

```diff
@@ -29,7 +29,8 @@ def table(connection_kwargs):
     yield name
 
     engine = create_engine(URL(**connection_kwargs))
-    engine.execute(f"DROP TABLE IF EXISTS {name}")
+    with engine.connect() as conn:
```

Comment:
https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#implicit-and-connectionless-execution-bound-metadata-removed